deploy/deploy-azure.ps1

```diff
@@ -44,6 +49,14 @@ param(
     # SKU for the Azure App Service plan
     $WebAppServiceSku = "B1",
 
+    [string]
+    # Azure AD cloud instance for authenticating users
+    $AzureAdInstance = "https://login.microsoftonline.com/",
+
+    [string]
+    # Azure AD tenant ID for authenticating users
+    $AzureAdTenantId = "common",
```

Comment:
As discussed during bug bash, we might want to not put "common" as the default and force users to provide a tenant

Reply:
Yup, I am currently going through and changing all of these